app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h2>Project Name: <%= @project.name %></h2>
+<h2>Project Material: <%= @project.material %></h2>
+<h2>Theme: <%= @project.challenge.theme %></h2>
+<h2>Number of Contestants: <%= @project.contestants.count %></h2>
```

Comment:
We don't want to use `.count` in the view either -- this should be calling whatever model methods you create to encapsulate the count and avg_experience methods -- so maybe @project.contestant_count and @project.avg_experience